### PR TITLE
[zh] sync /ingress/kubernetes-ingress/index.md

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with en.
```
content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md
```
- [x] Docs

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ test: yes
```
Comment (yulng) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:19]: 请按照[准备工作](/zh/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin)和[确定 Ingress IP 和端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)部分中的[Ingress Gateways 任务](/docs/tasks/traffic management/ingress control/）说明进行操作。

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ owner: istio/wg-networking-maintainers
...
 
-此任务描述如何使用 [Ingress 资源](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)为 Istio 配置入口网关以暴露网格内的服务。
+此任务描述如何使用 Kubernetes [Ingress 资源](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/)为 Istio 配置入口网关以暴露服务网格集群内的服务。
 
 {{< tip >}}
-建议使用 [Istio Gateway](/zh/docs/tasks/traffic-management/ingress/ingress-control/) 而不是 Ingress 来利用 Istio 提供的完整功能集，例如丰富的流量管理和安全功能。
+建议使用 [Gateway](/zh/docs/tasks/traffic-management/ingress/ingress-control/) 而不是 Ingress 来利用 Istio 提供的完整功能集，例如丰富的流量管理和安全功能。
 {{< /tip >}}
 
 ## 准备工作{#before-you-begin}
```
Comment (yulng) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:16]: Follow the instructions in the [Before you begin](/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin) and [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) sections of the [Ingress Gateways task](/docs/tasks/traffic-management/ingress/ingress-control/).
请按照[准备工作](/zh/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin)和[确定 Ingress IP 和端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)部分中的[Ingress Gateways 任务](/docs/tasks/traffic management/ingress control/）说明进行操作。

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md (hunk 1 above)
Comment (yulng) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:19]: Follow the instructions in the [Before you begin](/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin) and [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) sections of the [Ingress Gateways task](/docs/tasks/traffic-management/ingress/ingress-control/).
请按照[准备工作](/zh/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin)和[确定 Ingress IP 和端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)部分中的[Ingress Gateways 任务](/docs/tasks/traffic management/ingress control/）说明进行操作。

Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:19]: The original text is correct. You can check this link to learn more:
https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ test: yes
 
 ## 使用 Ingress 资源配置入口网关 {#configuring-ingress-using-an-ingress-resource}
 
-[Kubernetes Ingress 资源](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)公开了从集群外到集群内服务的 HTTP 和 HTTPS 路由。
+[Kubernetes Ingress 资源](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/)公开了从集群外到集群内服务的 HTTP 和 HTTPS 路由。
```
Comment (zirain) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:22]:
```suggestion
[Kubernetes Ingress](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/)公开了从集群外到集群内服务的 HTTP 和 HTTPS 路由。
```

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 4]
```diff
@@ -7,10 +7,10 @@ owner: istio/wg-networking-maintainers
 test: yes
 ---
 
-此任务描述如何使用 [Ingress 资源](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)为 Istio 配置入口网关以暴露网格内的服务。
+此任务描述如何使用 Kubernetes [Ingress 资源](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/)为 Istio 配置入口网关以暴露服务网格集群内的服务。
```
Comment (zirain) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:10]:
```suggestion
此任务描述如何使用 [Kubernetes Ingress](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/)为 Istio 配置入口网关以暴露服务网格集群内的服务。
```

**Review by zirain — APPROVED** [review]

**Pull request merged**